### DISCUSSION TITLE
Any interest in an Xbox One/Series port for devmode?

### DISCUSSION THREAD

**Discussion by ursusworks** [diasurgical/DevilutionX#1008] [Ideas]
Would be a nice addition to the underused space that is "homebrew" on devmode

**Comment by AJenbo** [comment]
Definatly

**Reply by ursusworks** [reply]
Hm...

Had a few goes on this last couple of days. But I think it's probably a few too many steps above my skill level.
Took the master. Created a new c++ uwp project for xbox. Import all the headers and source files into that project
Took SDL2 UWP from the hode project (heart of darkness) and the started to fix all the head references. But either this approach is wrong or I'm just not knowledgeable enough on eiter c++ or UWP (would not surprise me at all)
